gremlin-javascript/src/main/javascript/gremlin-javascript/test/unit/graphson-test.js

```diff
@@ -46,6 +46,30 @@ describe('GraphSONReader', function () {
       assert.strictEqual(result, item[1]);
     });
   });
+  it('should parse GraphSON Nan from GraphSON', function () {
+      const reader = new GraphSONReader();
+      var result = reader.read({
+                "@type": "g:Double",
+                "@value": "NaN"
+              });
+      assert.ok(isNaN(result));
+  });
+  it('should parse GraphSON -Infinity and Nan from GraphSON', function () {
```

Comment:
Just a minor nit but why does it say _and Nan_ when only `-Infinity` is tested here?

Reply:
just missed the capitlization in that string. i can fix it